src/me/mrCookieSlime/Slimefun/listeners/AncientAltarListener.java

```diff
@@ -37,6 +37,8 @@
 
 public class AncientAltarListener implements Listener {
 
+	public static boolean altarinuse = false;
```

Comment:
This restricts the maximum amount of Altars on the server to 1.
Use a HashSet with Locations instead to make this variable be per-altar.
And maybe move it to the Variables.class

Reply:
You sure about that?  I just tried making a second one and it works fine too...

Reply:
Yes. You use the same value for all altars.
You can exploit your "fix" by just activating one after another, waiting for one to finish, that sets it to false, allowing abuse of the other one that is still in use, but the value is set to false now.

Use a Location HashSet instead.

Reply:
Or... does this mean only one can be "in use" at any given time... as I've virtually no ability to test that.  I'll have to see how this is done elsewhere, I think I saw something like the Jet Boots stored a speed somehow, maybe I can use that as a starting point.

Reply:
alright killing this so the rest can go through... i'll work on using the location hashset after. reputting these files as undone.